_posts/2020-04-15-Cleard.md

```diff
@@ -0,0 +1,60 @@
+---
+author: John
+title: Week four co-winner Get Cleard, works to improve global screening processes
+---
+
+This week’s competition resulted in a tie- a first for our Codevid-19 global hackathon. Jaime Wood kindly offered to answer a few questions about her team’s entry, [Get Cleard](https://www.cleard.ai/).
+
+![Get Cleard](/images/blog/Get Cleard.png)
```

Comment:
Darn it, looks like we don't resize images properly yet.  I don't think I have the bandwidth to make the SASS edits required, so @johnny-newton can you replace this Markdown with HTML:

    <img src="/images/blog/Get Cleard.png" alt="Get Cleard" style="max-width: 100%;">

It'll fix this abomination:
<img width="1433" alt="image" src="https://user-images.githubusercontent.com/6334517/79393700-1fb16300-7f33-11ea-9ea4-d6c7a1f97edc.png">